source/renderer/app/components/wallet/WalletRestoreDialog.js

```diff
@@ -474,6 +505,59 @@ export default class WalletRestoreDialog extends Component<Props, State> {
           />
         )}
 
+        {!this.isCertificate() && this.isYoroi() && (
+          <RadioSet
+            label={intl.formatMessage(messages.recoveryPhraseTypeLabel)}
+            items={[
+              {
+                key: WALLET_RESTORE_TYPES.YOROI_LEGACY,
+                label: (
+                  <Fragment>
+                    15
```

Comment:
@DeeJayElly please replace all these hardcoded "number of words" with constants from here: https://github.com/input-output-hk/daedalus/blob/ff8b4559c2bc5473bbb5a322a1f626c7d92307a9/source/renderer/app/config/cryptoConfig.js
(please do that for ALL of them - 12, 15 & 27 words accross this file - and anywhere else where they might still be hardcoded. Thanks!

Reply:
Fixed